pkg/services/sqlstore/apikey.go

```diff
@@ -38,6 +40,21 @@ func (ss *SQLStore) GetAPIKeys(ctx context.Context, query *models.GetApiKeysQuer
 	})
 }
 
+// GetAPIKeys queries the database based
+// on input on GetApiKeysQuery
+func (ss *SQLStore) GetNonServiceAccountAPIKeys(ctx context.Context) []*models.ApiKey {
+	result := make([]*models.ApiKey, 0)
+	err := ss.WithDbSession(ctx, func(dbSession *DBSession) error {
+		sess := dbSession. //CHECK how many API keys do our clients have?  Can we load them all?
```

Comment:
There is no reliable way to check that, we don't have access to our customer's environments and even for the ones that send telemetrics, they're often more than half a year behind the release cadence.

Reply:
My assumption is that there wouldn't be many more API keys than users, so running the update in one batch should be ok.